install/v0.5.0-rc0/kfserving.yaml

```diff
@@ -16041,7 +16041,7 @@ data:
             "supportedFrameworks": [
               "sklearn"
             ],
-            "multiModelServer": "ture"
+            "multiModelServer": "true"
```

Comment:
scary (also done manually)

Reply:
It was in a RC release and we fixed later